src/components/PDOKAutoSuggest/PDOKAutoSuggest.test.tsx

```diff
@@ -67,6 +68,34 @@ describe('components/PDOKAutoSuggest', () => {
     })
   })
 
+  describe('fieldType', () => {
+    it('should call fetch with field type', async () => {
+      await renderAndSearch('Dam', { fieldType: 'adres' })
+      expect(fetch).toHaveBeenCalledWith(
+        expect.stringContaining(`${fieldTypeQs}(adres)`),
+        expect.objectContaining({ method: 'GET' })
+      )
+    })
+
+    it('should work with multiple field types', async () => {
+      await renderAndSearch('Dam', { fieldType: 'adres weg' })
+      expect(fetch).toHaveBeenCalledWith(
+        expect.stringContaining(`${fieldTypeQs}(adres weg)`),
+        expect.objectContaining({ method: 'GET' })
+      )
+    })
+
+    it('should ignore an empty string', async () => {
+      await renderAndSearch('Dam', {
+        fieldType: '',
+      })
+      expect(fetch).toHaveBeenCalledWith(
+        expect.not.stringContaining(fieldTypeQs),
+        expect.objectContaining({ method: 'GET' })
+      )
+    })
+  })
+
```

Comment:
We should also test the response. So in the first test case, addresses are returned, and in the second case addresses and roads.